src/main/kotlin/it/pagopa/aca/services/AcaService.kt

```diff
@@ -73,18 +75,25 @@ class AcaService(
                                 )
                             }
                     } else {
-                        gpdClient.updateDebtPosition(
-                            paFiscalCode,
-                            iupd.value(),
-                            acaUtils.updateOldDebitPositionObject(
-                                debitPosition,
-                                newDebtPositionRequestDto,
-                                iupd,
-                                newDebtPositionRequestDto.iban,
-                                newDebtPositionRequestDto.companyName,
-                                newDebtPositionRequestDto.postalIban
-                            )
-                        )
+                        creditorInstitutionClient
```

Comment:
it seems that this chunk of code is duplicated, could be refactored with a extract method?